### PR TITLE
Hacky fix for darwin

### PR DESCRIPTION
**Pull request by bbuscarino**
For now, I just rebased onto latest master until I have time to mess with this more. I have long layovers for travel today, so I'll commit to debugging the failures then.

### CODE REVIEW

**Comment by bbuscarino** [comment]
@gytis-ivaskevicius Finally got around to fixing this up, ready for review.

**Comment by gytis-ivaskevicius** [comment]
Thanks <3

File: lib/mkFlake.nix [hunk 1]
```diff
@@ -161,7 +161,7 @@ let
                           config = selectedNixpkgs.config // config.nixpkgs.config;
                         } // { inherit (selectedNixpkgs) name input; };
                 }
-              else { _module.args.pkgs = selectedNixpkgs; })
+              else { })
```
Comment (gytis-ivaskevicius) [inline lib/mkFlake.nix:164]: Is this change necessary? i feel like this would break something

**Review by gytis-ivaskevicius — APPROVED** [review]

File: lib/mkFlake.nix (hunk 1 above)
Comment (bbuscarino) [reply lib/mkFlake.nix:164]: I don't remember why exactly this was necessary, I want to look into it later tonight. The tests were green though so...?
